src/test/java/seedu/address/testutil/TypicalLocations.java

```diff
@@ -65,8 +65,21 @@ public static LocationBook getTypicalLocationBook() {
         return lb;
     }
 
+    public static LocationBook getTypicalLocationBookForTest() {
+        LocationBook lb = new LocationBook();
+        for (Location location : getLocationsForTest()) {
+            lb.addLocation(location);
+        }
+        return lb;
+    }
+
```

Comment:
This method seems very similar to the above method getTypicalLocationBook. I feel that since both methods are used in test cases, naming this method getTypicalLocationBookForTest doesnt differentiate the purpose of the 2 methods. Perhaps naming it as getUnorderedTypicalLocationBook() would be better?

Reply:
Done.